### DISCUSSION TITLE
ACM to ACF Migration

### DISCUSSION THREAD

**Discussion by manish85sixty** [wpengine/atlas-content-modeler#685] [Q&A / Help / Support]
As Atlas Content Modeler is entering an end-of-life phase, Is there any way to covert the ACM schema JSON to ACF schema JSON to migrate from ACM to ACF?

**Comment by mindctrl** [comment]
@manish85sixty we don't have a tool for converting or migrating. We have some documentation here to help. https://github.com/wpengine/atlas-content-modeler/blob/main/docs/end-of-life/index.md

Happy to answer any questions you have related to the migration.

**Comment by manish85sixty** [comment]
@mindctrl We have lots of content modeler and each have lots of field. So, we will have to create everything manually in ACF? Also, Can we have the same custom post and field groups names in ACF otherwise we will have to make lots of changes in frontend GraphQl request?

Thanks,
Manish

**Comment by mindctrl** [comment]
@manish85sixty you can use the same post and field group names in ACF that you use in ACM. If you do that, all your data will show up without needing to copy it to new field names. Once you have that in place, you'll need to update any GraphQL queries in your app to match the schema that the WPGraphQL for ACF plugin provides. It's not a lot different, but it outputs the fields nested underneath a fields node. In the doc linked above, you can see the subtle difference in the example query.

**Comment by NazarTikhoniuk** [comment]
Greetings @mindctrl 
Do you have any recommendations on how to migrate Relationship field type data?
